handwrite/cli.py

```diff
@@ -74,6 +75,12 @@ def main():
     parser.add_argument(
         "--config", help="Use custom configuration file/directory", default=None
     )
+    parser.add_argument("--filename", help="Font File name", default=None)
+    parser.add_argument("--family", help="Font Family name", default=None)
+    parser.add_argument("--style", help="Font Style name", default=None)
```

Comment:
What's the difference between filename and fontname? When we install the font, would it just take the filename when installed?

Reply:
Font name is family name + style name (the font will show up as this name in the font list when loaded along with the other default fonts available) 
Filename is the name of .ttf file when the font is downloaded.

Reply:
Got it